resources/src/main/java/org/robolectric/res/android/CppAssetManager2.java

```diff
@@ -774,7 +773,7 @@ private ApkAssetsCookie FindEntry(int resid, short density_override,
   public boolean GetResourceName(int resid, ResourceName out_name) {
     final Ref<FindEntryResult> entryRef = new Ref<>(null);
     ApkAssetsCookie cookie =
-        FindEntry(resid, (short) 0 /* density_override */, true /* stop_at_first_match */, entryRef);
+        FindEntry(resid, (short) 0 /* density_override */, /* stop_at_first_match */ entryRef);
```

Comment:
This comment should be deleted if we decide to remove `stop_at_first_match` from `FindEntry`. We also should discuss whether we should delete `stop_at_first_match` firstly.

Reply:
Regarding the param `stop_at_first_match`, we can delete it since we are not using it anywhere, If we can delete it then I will push again the changes with deleted comment as well

Reply:
Okay, I can accept to remove this field. Hi @hoisie , what do you think about it?